### PR TITLE
Feature toggle work allocation and change some parameters

### PR DESCRIPTION
**Pull request by udaydenduluri33**
JIRA link (if applicable)
https://tools.hmcts.net/jira/browse/EUI-3548

Change description
Integrating with WA/ EXUInode api

Does this PR introduce a breaking change? (check one with "x")

[ ] Yes
[X] No

Original PR to master is https://github.com/hmcts/ccd-case-ui-toolkit/pull/715

### CODE REVIEW

**Review by Mo-Lala84 — APPROVED** [review]
lgtm

File: src/shared/components/case-editor/services/work-allocation.service.ts
Comment (paul-graham) [inline src/shared/components/case-editor/services/work-allocation.service.ts:43]: This means that if `error` is not defined, we'll throw it...?

File: src/shared/components/case-editor/services/work-allocation.service.ts
Comment (paul-graham) [inline src/shared/components/case-editor/services/work-allocation.service.ts:39]: `error.status` (the second condition) is unnecessary considering the third and fourth conditions).

**Review by paul-graham — COMMENTED** [review]
This appears to have been merged with a single approver while I was in the middle of doing a review.

**Comment by udaydenduluri33** [comment]
Just a fyi - Me and @rayliangatos  were on call before we merged this. He verbally agreed to merge it. And I merged it

**Comment by rayliangatos** [comment]
> Just a fyi - Me and @rayliangatos were on call before we merged this. He verbally agreed to merge it. And I merged it

Yes I verbaly ask Uday to merge to my PR, I think the approvals will be cleared after new commits in? And this can be reviewed againts the new PR.

**Comment by paul-graham** [comment]
> > Just a fyi - Me and @rayliangatos were on call before we merged this. He verbally agreed to merge it. And I merged it
> 
> Yes I verbaly ask Uday to merge to my PR, I think the approvals will be cleared after new commits in? And this can be reviewed againts the new PR.

Going forward, can we please make sure approvals are recorded on the PR prior to a merge, not given verbally. There's no way to grant an approval on the PR after the fact.

**Comment by rayliangatos** [comment]
> > > Just a fyi - Me and @rayliangatos were on call before we merged this. He verbally agreed to merge it. And I merged it
> > 
> > 
> > Yes I verbaly ask Uday to merge to my PR, I think the approvals will be cleared after new commits in? And this can be reviewed againts the new PR.
> 
> Going forward, can we please make sure approvals are recorded on the PR prior to a merge, not given verbally. There's no way to grant an approval on the PR after the fact.

My branch is not a protected branch, so I don't think strict rule(2 approvals) needs to be applied here. In another word I think the hotfix/EUI-3594-error-message-display is acted as a protected release branch as it's a LIVE version.

**Comment by udaydenduluri33** [comment]
> > > Just a fyi - Me and @rayliangatos were on call before we merged this. He verbally agreed to merge it. And I merged it
> > 
> > 
> > Yes I verbaly ask Uday to merge to my PR, I think the approvals will be cleared after new commits in? And this can be reviewed againts the new PR.
> 
> Going forward, can we please make sure approvals are recorded on the PR prior to a merge, not given verbally. There's no way to grant an approval on the PR after the fact.

But In principle yes

**Comment by paul-graham** [comment]
> > > > Just a fyi - Me and @rayliangatos were on call before we merged this. He verbally agreed to merge it. And I merged it
> > > 
> > > 
> > > Yes I verbaly ask Uday to merge to my PR, I think the approvals will be cleared after new commits in? And this can be reviewed againts the new PR.
> > 
> > 
> > Going forward, can we please make sure approvals are recorded on the PR prior to a merge, not given verbally. There's no way to grant an approval on the PR after the fact.
> 
> My branch is not a protected branch, so I don't think strict rule(2 approvals) needs to be applied here. In another word I think the hotfix/[EUI-3594](https://tools.hmcts.net/jira/browse/EUI-3594)-error-message-display is acted as a protected release branch as it's a LIVE version.

But there's a version getting released and going live from your branch so, whilst it's not "protected", it should be treated as such.

**Pull request merged**
